src/getCoinCombination.test.js

```diff
@@ -7,4 +7,44 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return [0, 0, 0, 0] for 0 ', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
+
+  it('If accept pennys only', () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+  });
+
+  it('If accept only nikel', () => {
```

Comment:
The test description 'If accept only nikel' should be revised for clarity and correct spelling. Consider changing it to 'should return [0, 1, 0, 0] for 5 cents (only nickels)'. The correct spelling is 'nickel'.